scripts/script.js

```diff
@@ -0,0 +1,162 @@
+let tasks = [];
+
+// Get Time and date in the required format  27/07/2022 @ 9:38 pm
+let getTime = () => {
+  let date = new Date();
+  let dd = date.getDate();
+  let mm = date.getMonth() + 1;
+  let yyyy = date.getFullYear();
+  let hours = date.getHours();
+  let minutes = date.getMinutes();
+  let ampm = hours >= 12 ? "pm" : "am";
+  hours = hours % 12;
+  hours = hours ? hours : 12;
+  minutes = minutes < 10 ? "0" + minutes : minutes;
+
+  if (dd < 10) {
+    dd = "0" + dd;
+  }
+  if (mm < 10) {
+    mm = "0" + mm;
+  }
+  return (
+    dd + "/" + mm + "/" + yyyy + " @ " + hours + ":" + minutes + " " + ampm
+  );
+};
+
+let getTasks = () => {
+  data = localStorage.getItem("tasks");
+  if (data !== null) tasks = JSON.parse(data);
+  return tasks;
+};
+
+// Add New Tasks
+let addTasks = () => {
+  tasks = getTasks();
+  let length = 0;
+  let id = 1;
+  if (tasks.length > 0) {
+    length = tasks.length;
+    id = tasks[length - 1].id;
+    id = id + 1;
+  }
+  let time = getTime();
+  tasks.push({
+    id: id,
```

Comment:
Instead of creating the length and id attribute above and adding that if loop, we could simply add one line over here. 
id: tasks.length > 0 ? tasks.length + 1 : 1;